plugins/@grouparoo/spec-helper/src/lib/workflows/import.ts

```diff
@@ -35,6 +35,7 @@ export async function ImportWorkflow() {
     }
 
     await specHelper.runTask("profiles:checkReady", {});
+    await specHelper.runTask("profiles:enqueueExports", {});
```

Comment:
It is good to see that after changing this helper here all the tests remain passing!